include/dronecore.h

```diff
@@ -30,17 +59,83 @@ class DroneCore
         CONNECTIONS_EXHAUSTED
     };
 
+    /**
+     * Translates the ConnectionResult enum to a human-readable English string.
+     */
     static const char *connection_result_str(ConnectionResult);
 
+    /**
+     * Adds a UDP connection with the default arguments.
+     *
+     * This will listen on UDP port 14540 on localhost.
+     *
+     * @return the result of adding the connection.
+     */
     ConnectionResult add_udp_connection();
+
+    /**
+     * Adds a UDP connection with a specific port number.
+     *
+     * @param local_port_number The local UDP port to listen to
+     * @return the result of adding the connection.
+     */
     ConnectionResult add_udp_connection(int local_port_number);
 
+    /**
+     * Return vector of device UUIDs.
+     *
+     * This returns a vector of the UUIS of all devices that have been discovered so far and can be
```

Comment:
1. Typo -UUIS = UUIDs?

2. Where it says "of all devices that have been discovered so far and can be accessed." is the bit "and can be accessed necessary?"

Reply:
Fixed in 8327f85.